testpilot/frontend/static-src/styles/modules/_experiment-details.scss

```diff
@@ -165,11 +232,11 @@
 }
 
 .eol-block {
-  padding: 10px !important;
-  border-radius: 3px;
   background: $eol-light-orange;
   border: 1px solid $eol-orange;
+  border-radius: 3px;
   margin-bottom: 20px;
+  padding: 10px !important;
```

Comment:
@pdehaan this is left over from @dannycoates...if you want to get on somebody's case about using important, take it up with him :)